README.md

```diff
@@ -1,57 +1,99 @@
-# Project Name
+---
+page_type: sample
+languages:
+- javascript
+products:
+- msal-react
+- ms-graph
+- azure-storage
+- azure-key-vault
+- azure-functions
+- azure-app-service
+- azure-active-directory
+- azure-active-directory-b2c
+- microsoft-identity-platform
+description: "Tutorial: Enable your React single-page application (SPA) to sign-in users and call APIs with the Microsoft identity platform"
```

Comment:
>Enable your React single-page application (SPA) to sign-in users and call APIs with the Microsoft identity platform [](start = 24, length = 115)

Proposing , "Integrate your React SPA to sign-in users , call the Graph API and deploy on Azure". We'd change the "Graph" to "Graph and protected APIs" when more chapters are added

Reply:
How's this?

Enable your React single-page application to sign-in users, call the Graph API and deploy on Azure